library/analytics/build.gradle

```diff
@@ -25,6 +25,8 @@ android {
 dependencies {
     implementation project(':library:base')
 
+    implementation "com.crashlytics.sdk.android:crashlytics:${deps.crashlytics}"
```

Comment:
you can remove this crashlytics dependency from here, it's not being used anymore :)